SwaMe.Desktop/AnalysisSettingsFileWriter.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using MzmlParser;
+
+namespace SwaMe.Desktop
+{
+    public class AnalysisSettingsFileWriter
+    {
+        public void WriteASFile(Run run, string dateTime, List<string> fileNames) //Multiple files
+        {
+            string originalDirectory = Directory.GetCurrentDirectory().ToString();
+            Directory.SetCurrentDirectory(Directory.GetParent(Directory.GetParent(originalDirectory).ToString()).ToString());
+            string ASFile;
+            ASFile = dateTime + "_AnalysisSettings_" + ".txt";
+            StreamWriter streamWriter = new StreamWriter(ASFile);
+            streamWriter.Write("Analysis Settings:");
+            streamWriter.Write(Environment.NewLine);
+            streamWriter.Write("InputFiles:");
+            streamWriter.Write(Environment.NewLine);
+
+            foreach (string file in fileNames) 
+            {
+                streamWriter.Write(Path.GetFileNameWithoutExtension(file));
+                streamWriter.Write(Environment.NewLine);
+            }
+
+            //write streamWriter
+            string[] phraseToWrite = { "MassTolerance", Convert.ToString(run.AnalysisSettings.MassTolerance), "RtTolerance", Convert.ToString(run.AnalysisSettings.RtTolerance),
+                                        "IrtLibrary",string.IsNullOrEmpty(Convert.ToString(run.AnalysisSettings.IrtLibrary)) ? "None" : Convert.ToString(run.AnalysisSettings.IrtLibrary), "IrtMassTolerance",Convert.ToString(run.AnalysisSettings.IrtMassTolerance),
+                                        "IrtMinIntensity",Convert.ToString(run.AnalysisSettings.IrtMinIntensity),"IrtMinPeptides",Convert.ToString(run.AnalysisSettings.IrtMinPeptides),
+                                        "CacheSpectraToDisk",Convert.ToString(run.AnalysisSettings.CacheSpectraToDisk),"MinimumIntensity",Convert.ToString(run.AnalysisSettings.MinimumIntensity),
+                                        "RunEndTime",Convert.ToString(run.AnalysisSettings.RunEndTime)};
```

Comment:
I know this is a copy of code elsewhere, but can I recommend x.ToString() rather than Convert.ToString(x)?  Also, in what locale should these be written, given the SA use of "," as a decimal separator and hence ";" as a "CSV" separator?